### PR TITLE
Add move and copy operations

### PR DESCRIPTION
**Pull request by ltwlf**
## Summary
Adds `move` and `copy` operations per updated JSON Atom spec (Section 6.6, 6.7).

- **Move**: atomically removes from source, adds at target. Inversion swaps paths.
- **Copy**: deep-clones from source to target. Optional `value` field for reversibility.
- Self-move prevention: `from` must not equal `path` or be a prefix of `path`.
- `fromAtom` throws for move/copy (v4 changeset format doesn't support two-path ops).
- 47 new tests covering validation, apply, inversion, sequential semantics, and edge cases.
- Bumps to `5.0.0-alpha.7`

## Test plan
- [x] 393 tests pass (346 existing + 47 new)
- [x] Lint and typecheck clean
- [x] Build succeeds

### CODE REVIEW

File: src/jsonAtom.ts [hunk 1]
```diff
@@ -86,6 +88,33 @@ export function validateAtom(atom: unknown): { valid: boolean; errors: string[]
...
+        if ('oldValue' in op) {
+          errors.push(`operations[${i}]: move operation must not have oldValue`);
+        }
+        if (typeof op.from === 'string' && typeof op.path === 'string') {
+          if (op.from === op.path) {
+            errors.push(`operations[${i}]: move operation from must not equal path (self-move)`);
+          }
+          if (op.from !== '$' && (op.path.startsWith(op.from + '.') || op.path.startsWith(op.from + '['))) {
+            errors.push(`operations[${i}]: move operation path must not be a subtree of from`);
+          }
```
[the remark this answers is not in the document]
Comment (ltwlf) [reply src/jsonAtom.ts]: Same as before — validation is best-effort on raw strings. Path normalization could be added later but the apply-time checks catch actual conflicts.

File: src/jsonAtom.ts [hunk 2]
```diff
@@ -557,6 +597,45 @@ export function invertAtom(atom: IJsonAtom): IJsonAtom {
...
+        const prop = seg.property;
+        const isPath = !seg.literalKey && prop.includes('.') && NESTED_PATH_RE.test(prop);
+        const arr = current as any[];
+        current = arr.find((el: any) => {
+          const resolved = isPath ? prop.split('.').reduce((c: any, s: string) => c?.[s], el) : el[prop];
+          return JSON.stringify(resolved) === JSON.stringify(seg.value);
+        });
+        break;
+      }
+      case 'valueFilter': {
```
[the remark this answers is not in the document]
Comment (ltwlf) [reply src/jsonAtom.ts]: Fixed in 015c0f6 — type guards throw descriptive errors for null/non-object property access and non-array filter/index access.
